src/models/attendee.js

```diff
@@ -61,6 +65,8 @@ const mapAttendeePropertyToAttendeeObject = (attendeeProperty) => {
 		role: attendeeProperty.role,
 		rsvp: attendeeProperty.rsvp,
 		uri: attendeeProperty.email,
+		// TODO Expose member attribute in calendar-js
```

Comment:
```suggestion
		// TODO Expose member attribute in calendar-js
```
easy workaround but propably going to fix this in calendar-js first